metadata/algolia/jobs.xml

```diff
@@ -1,63 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <jobs xmlns="http://www.demandware.com/xml/impex/jobs/2015-07-01">
 
-    <job job-id="AlgoliaProductsIndex" priority="0">
```

Comment:
Do we want to remove the old jobs outright without having a grace period where both jobs are in the cartridge for a time? If the decision is taken to remove them, we should go all in and remove them upon import with `mode="delete"`.